src/lib/signature_lib/private_key.ml

```diff
@@ -66,3 +64,7 @@ let of_yojson = function
           Error ("Signature_lib.Private_key.of_yojson: " ^ Exn.to_string exn) )
   | _ ->
       Error "Signature_lib.Private_key.of_yojson: Expected a string"
+
+let of_string_exn s = Inner_curve.Scalar.of_string s
+
+let to_string t = Inner_curve.Scalar.to_string t
```

Comment:
I personally don't see the value of aliasing a function. This only creates confusion. downstream user would always need to check the underlying implementation to ensure they're the same thing.

Unless this is used as part of functor calculation, I'd like to reject such change.